backend/package.json

```diff
@@ -13,7 +13,9 @@
       "start": "strapi start",
       "build": "strapi build && npm run build:addons",
       "build:addons": "cd src/plugins/addons && npm run build",
-      "strapi": "strapi"
+      "strapi": "strapi",
+      "type-check": "tsc --pretty --noEmit && npm run type-check:addons",
```

Comment:
do we need this `tsc --pretty --noEmit` if it just gets called in `/addons` `type-check`?

Reply:
The first `tsc` uses `backend/tsconfig.json` to check many of the files in `backend`, so I think it's good to have it. The `addons` plugin is technically it's own package with it's own `tsconfig.json`, so that's why we're calling it separately here.